gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/util/EhrResourceExtractorUtil.java

```diff
@@ -54,6 +56,34 @@ public static boolean hasEhrFolder(RCMRMT030101UK04Component component) {
         return component.getEhrFolder() != null;
     }
 
+    public static RCMRMT030101UK04EhrComposition extractEhrCompositionForNarrativeStatement(RCMRMT030101UK04EhrExtract ehrExtract,
```

Comment:
is this method and the following extractValidDocumentReferenceEhrCompositions method called anymore? Im not sure they even should as they arent handling nested Narrative Statements?

Reply:
> is this method and the following extractValidDocumentReferenceEhrCompositions method called anymore? Im not sure they even should as they arent handling nested Narrative Statements?

yea they arent being used so they can be removed